public/components/trace_analytics/components/traces/traces_content.tsx

```diff
@@ -73,6 +73,10 @@ export function TracesContent(props: TracesProps) {
   const [includeMetrics, setIncludeMetrics] = useState(false);
   const isNavGroupEnabled = coreRefs?.chrome?.navGroup.getNavGroupEnabled();
 
+  useEffect(() => {
```

Comment:
Can this useEffect be merged with the one on line number: `95` https://github.com/opensearch-project/dashboards-observability/pull/2257/files#diff-102a88265e317b8fd409fdb0aafb158beb0a68bec9b93f971e49ee8965fca62fR95-R122

Reply:
Yes added the new dependencies to the use effect
```
    startTime,
    endTime,
    props.dataSourceMDSId,
```
This triggers the refresh